Nos/Service/Relay/PagedRelaySubscription.swift

```diff
@@ -72,7 +72,7 @@ class PagedRelaySubscription {
                     }
                           
                     newUntilDates[subscription.relayAddress] = newDate
-                    await subscriptionManager.decrementSubscriptionCount(for: subscriptionID)
+                    await relayService.decrementSubscriptionCount(for: subscriptionID)
```

Comment:
```suggestion
                   relayService.decrementSubscriptionCount(for: subscriptionID)
```
<img width="667" alt="Screenshot 2024-06-03 at 4 29 20 PM" src="https://github.com/planetary-social/nos/assets/59564/35686832-713d-4015-9809-4c2d4cc62554">